policy/framework.go

```diff
@@ -374,6 +374,7 @@ func (f *Framework) updateAllChecksums(ctx context.Context,
 		}
 
 		executionChecksum = executionChecksum.AddUint(uint64(group.Type))
+		executionChecksum = executionChecksum.AddUint(uint64(group.ReviewStatus))
```

Comment:
do we have tests for checksums? would be nice to add that there

Reply:
we don't but I can try to add one that checks that the checksum is different when the review status changes

Reply:
+1, we should build up the list of test cases here, a minimal (just review status change) start is great

Reply:
tests 🎉